frontend/pages/courses_id_review/components/ProblemBeingSolved/components/Subheader/index.js

```diff
@@ -41,12 +41,13 @@ class Subheader extends React.Component {
       />
     </div>
 
+
   renderSimulatedReview = () =>
     <section className={`Subheader ${css.section} -simulated-review`}>
       <div className="container">
         <div className="instructions -desktop">
-          <p><em className="yellow-emphasis">Test drive</em> - results will not be recorded. Press ENTER to reveal answers</p>
-        </div>
+          <p><em className="yellow-emphasis">Test drive</em> - results will not be recorded. Press ENTER to reveal answers<span class="span_align"> {this.renderVolumeButton()}</span></p>
```

Comment:
We have syntax `.span-align` for css classNames.
Besides, we don't need this `span` wrapper, we can just add `. Subheader.-simulated-review .volume-button{}` style.

Reply:
There is no prop `class` in react, it's `className`.